evm/vm/flavors/mainnet/__init__.py

```diff
@@ -8,7 +8,7 @@
 )
 
 
-MainnetEVM = EVM.configure(
+MainnetEVM = Chain.configure(
```

Comment:
I was assuming this was done, but if not, yes, the term `EVM` should likely no longer be present in the codebase unless it is referring specifically the EVM concept (and not the `EVM` object).